mimic/model/clb_objects.py

```diff
@@ -78,6 +104,14 @@ def add_load_balancer(self, tenant_id, lb_info, lb_id, current_timestamp):
 
         return {'loadBalancer': new_lb}, 202
 
+    def set_attributes(self, lb_id, kvpairs):
+        """
+        Sets zero or more attributes on the load balancer object.
+        Currently supported attributes include: status.
+        """
+        if "status" in kvpairs:
```

Comment:
I am concerned that if we silently ignore any values other than `"status"` it is going to be fiendishly difficult to debug misspellings on the client side.  I would suggest adding a check to make sure that `kvpairs` is a dictionary containing *only* supported keys (i.e. exclusively `"status"` at this moment, but presumably expanded in the near future?) and then after having performed the check, simply doing `self.lbs[lb_id].update(kvpairs)`.